token/cli/src/clap_app.rs

```diff
@@ -669,6 +672,16 @@ pub fn app<'a, 'b>(
                             "Specify address that stores token metadata."
                         ),
                 )
+                .arg(
+                    Arg::with_name("group_address")
+                        .long("group-address")
+                        .value_name("ADDRESS")
+                        .takes_value(true)
+                        .conflicts_with("enable_group")
+                        .help(
+                            "Specify address that stores token group configurations."
+                        ),
+                )
```

Comment:
This is also an issue with the `metadata-address` arg, but this arg should have `.validator(is_valid_pubkey)`, otherwise it's possible to put it an invalid address here, which will have mint creation proceed without the group pointer and without throwing an error

Reply:
Done. Also opened a PR for metadata. #5926